client/src/__tests__/__snapshots__/container.test.js.snap

```diff
@@ -0,0 +1,53 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should render correctly with no props 1`] = `
```

Comment:
Does Jest Snapshot give you all of this information when you run the test?

Reply:
yes this file gets generated